docker/polkadot-collator_builder.Containerfile

```diff
@@ -1,36 +1,36 @@
 # This file is sourced from https://github.com/paritytech/polkadot/blob/master/scripts/ci/dockerfiles/polkadot/polkadot_builder.Dockerfile
```

Comment:
I think this file needs to be renamed too. It still says `polkadot-collator...` but is referenced in multiple places as `polakdot-parachain...`.

Reply:
you are right, good catch

Reply:
fixed